Class/CleanupVMs.ps1

```diff
@@ -5,66 +5,99 @@ param
     [string] $LabName,
 
     # Credential path
-    [Parameter(Mandatory=$false, HelpMessage="Path to file with Azure credentials")]
-    [string] $credentialPath = "$env:APPDATA\AzProfile.txt"
+    [Parameter(Mandatory=$false, HelpMessage="Path to file with Azure Profile")]
+    [string] $profilePath = "$env:APPDATA\AzProfile.txt"
 )
 
 # Stops at the first error instead of continuing and potentially messing up things
-$global:erroractionpreference = 1
+#$global:erroractionpreference = 1
```

Comment:
Maybe I understand it now. Is it so that you can capture errors for all jobs instead of just the first one? But, isn't that already accomplish by the try ... catch in line 54?